contracts/contracts/NoLossDao.sol

```diff
@@ -270,19 +269,21 @@ contract NoLossDao is Initializable {
   function delegateVoting(address delegatedAddress)
     public
     userStaked(msg.sender)
-    userHasNoActiveProposal(msg.sender)
+    userHasNoActiveProposal(msg.sender) //Careful when in cooldown can delegate then ???
```

Comment:
This should be `userIsNotBeneficiary` because we don't want this function to work at all if they are a beneficiary (who created a project).

Reply:
The kind of things I'm nervous as being exploitable are things where someone changes their role somehow and then gives themselves extra privilege from the past somehow. I don't see how now, but it is what I'm most wary about.